README.md

```diff
@@ -5,70 +5,58 @@ entity resolution (ER) pipelines.
 
 ## Installation
 
-- Install docker
-    - Mac: https://docs.docker.com/desktop/install/mac-install/
-    - Windows: https://docs.docker.com/desktop/install/windows-install/
-- Clone and install this repo
-```
-$ cd <path/to/repositories/>
-$ git clone git@github.com:ihmeuw/easylink.git
-$ # OR `git clone https://github.com/ihmeuw/easylink.git`
-$ cd easylink
-$ pip install .
-```
+There are a few things to install in order to use this package:
 
-## Running a pipeline
+- Install singularity. If this is not already installed on your system, you will
```

Comment:
@zmbc @patricktnast @rmudambi Is this good enough? I don't personally have a handle on what exactly the singularity requirements are. Like, we currently are using `singularity-ce version 4.1.4` on the cluster. Would some other version work? What about non-ce singularity? 🤷